src/components/photoGallery/PhotoGallery.scss

```
@@ -0,0 +1,12 @@
+.image-gallery {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    justify-content: space-around;
+    gap: 10px;
+}
+.space-image {
```

Line between CSS definitions
```suggestion
}

.space-image {
```